addons/website_blog/models/website_blog.py

```diff
@@ -128,8 +128,14 @@ def _compute_website_url(self):
             blog_post.website_url = "/blog/%s/%s" % (slug(blog_post.blog_id), slug(blog_post))
 
     def _default_content(self):
-        return '''
-            <p class="o_default_snippet_text">''' + _("Start writing here...") + '''</p>
+        return f'''
+            <section class="s_text_block pt16 pb16" data-snippet="s_text_block">
```

Comment:
Did you investigate the possibility of using the text block snippet direct here? (= rendering the snippet with a custom text, to avoid duplication).